types/cidr-matcher/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for cidr-matcher 2.1
+// Project: https://github.com/pracucci/node-cidr-matcher
+// Definitions by: Sam Murphy <https://github.com/SamMurphyDev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+declare class Matcher {
+    /**
+     * Constructs a matcher for a list of CIDR ranges
+     *
+     * @param cidrRanges - list of IPv4 & IPv6 CIDR ranges
+     */
+    constructor(cidrRanges: readonly string[]);
```

Comment:
can this param be optional?
https://github.com/pracucci/node-cidr-matcher/blob/master/src/Matcher.js#L5

Reply:
Yes, it can be. Updated and tested implementation.